### ISSUE TITLE
Evaluation Metrics Usage for Multi-reference

### ISSUE THREAD

**Issue by cs329yangzhong** [eujhwang/meme-cap#3]
Dear authors, thanks for sharing this exciting work. I found the dataset super helpful. One question I have is on the evaluation: given that you collect more than one human-annotated explanation per meme, how do you compute the ROUGE and BERTScores? Did you run the VL model once to generate the candidate summary, compare it with each human annotations, and report the average?

**Comment by eujhwang** [comment]
Hello, thank you for the good question!

We run the VL model once to generate the candidate summary.
For ROUGE score calculation: we compare the generation with each human annotation and take the best score. Then average scores for all images.
We used and followed this: https://github.com/salaniz/pycocoevalcap

For BERTScore: we compare the generation with each human annotation and take the best score. Then average scores for all images.
We used and followed this: https://pypi.org/project/bert-score/

Hope this helps!
